### DISCUSSION TITLE
Airgap update from 2.21 to 2.30 issue

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#3243] [Unsupported Versions]
I am running a distributed test grid in an ESXi 6.7 environment, with plenty of resources.

I was attempted to update with SOUP after mounting the new ISO to the manager. Update appeared to go well, but after ~30 minutes, the version numbers showing in grid did not change, and neither did the SecurityOnion/VERSION. I rebooted the server to see if it was necessary (although I suspect not). After ~15m got the following

![Capture2](https://user-images.githubusercontent.com/12967094/109742205-a9463f80-7b9c-11eb-8519-ef3b57210a68.PNG)

Again, this is a test grid, so it's not urgent, but if you could please advise on which logs to investigate I would appreciate it.

Thanks for all the great work you all do!

**Comment by TOoSmOotH** [comment]
What does the soup log say in /root?

**Reply by darth-drew** [reply]
Looks like that log got overwritten when I tried to run soup again.
![image](https://user-images.githubusercontent.com/12967094/109818380-ecd29500-7c00-11eb-9d59-94805c9c3d81.png)

**Reply by TOoSmOotH** [reply]
Can you run:

`sudo docker images`

**Reply by darth-drew** [reply]
![image](https://user-images.githubusercontent.com/12967094/109820385-ef35ee80-7c02-11eb-80aa-0982f6086655.png)

**Reply by TOoSmOotH** [reply]
can you do a df -h /

**Reply by darth-drew** [reply]
![image](https://user-images.githubusercontent.com/12967094/109821374-e85bab80-7c03-11eb-938d-ea3bd5696e1b.png)
200GB capacity on Thick provision Lazy Zeroed disk. Not sure if that is relevant

**Reply by TOoSmOotH** [reply]
One last check.. Can you do this:

`ls -la /nsm/docker-registry/docker/registry/v2/repositories/security-onion-solutions/so-logstash/_manifests/tags`

If that does not show any 2.3.30 tags then modify /etc/soversion to be 2.3.21 again and rerun soup

**Reply by darth-drew** [reply]
Thanks for your time Mike.
![image](https://user-images.githubusercontent.com/12967094/109823659-21951b00-7c06-11eb-82d2-67e629298190.png)
 Still thinking we should make the change?

**Reply by TOoSmOotH** [reply]
can you run a salt-call state.highstate then?

**Reply by darth-drew** [reply]
still running but I am seeing 404 errors on so-filebeat/elastalert/etc.. manifest not found.

**Reply by darth-drew** [reply]
![image](https://user-images.githubusercontent.com/12967094/109825626-01665b80-7c08-11eb-9134-2ef7840e649c.png)
Seems like a lot of docker errors.

**Reply by darth-drew** [reply]
Mike, Are you hoping to learn something from this on your alls end? I'd keep it around, but I could also just rebuild the grid. I don't have enough resources at my disposal to do both unfortunately.

**Reply by TOoSmOotH** [reply]
You can go ahead and kill it. I am not able to recreate on my end.

**Reply by darth-drew** [reply]
Thanks for sharing some troubleshooting steps. I learned some things.
